src/platform/user/profile/constants/backendServices.js

```diff
@@ -20,5 +20,9 @@ export default {
 
   // Core Form Features
   SAVE_IN_PROGRESS: 'form-save-in-progress',
-  FORM_PREFILL: 'form-prefill'
+  FORM_PREFILL: 'form-prefill',
+
+  // 526 beta
+  CLAIM_INCREASE: 'claim_increase',
+  CLAIM_INCREASE_AVAILABLE: 'claim-increase-available'
```

Comment:
Are these defined just on the front end?

Also, what's the difference between the two?

Reply:
No, they're on the backend, too. The first one is for the beta, the second one is to denote if we can still submit increase applications at the moment (it'll get removed after we hit our limit for the day/week)

Reply:
Ahhh, cool. :+1:
Might make sense to document that somewhere too since they seem pretty similar just looking at the names.